libkpod/config.go

```diff
@@ -0,0 +1,63 @@
+package libkpod
+
+import (
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+	cstorage "github.com/containers/storage"
+)
+
+// Config stores the configuration options for libkpod
+type Config struct {
```

Comment:
Probably want to have TOML field name annotations to match CRI-O's config. Most of the options should be the same, so the same names/layout should be sufficient.

Eventually we can deduplicate and have CRI-O's config be a superset of this config.